wsagent/che-core-api-git/src/main/java/org/eclipse/che/api/git/exception/GitInvalidRefNameException.java

```diff
@@ -0,0 +1,51 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2015 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.api.git.exception;
+
+/**
+ * Git Invalid ref spec name Exception
```

Comment:
Do you think that this javadoc is good? I prefer to leave class without javadoc instead of writing something like that. Please rewrite it or remove